src/ConfigParser.cpp

```diff
@@ -44,8 +44,21 @@ ConfigSettings ConfigParser::extractStreamInfo() {
 void ConfigParser::setBrokers(std::string const &Brokers,
                               ConfigSettings &Settings) {
   Settings.Brokers.clear();
-  auto a = split(Brokers, ",");
-  for (auto &x : a) {
+  std::vector<std::string> ret;
```

Comment:
`split` should still be a separate function for readability+maintainability. It can just be a free function in this file (`ConfigParser.cpp`).